atom/browser/net/atom_network_delegate.cc

```diff
@@ -431,25 +431,25 @@ bool AtomNetworkDelegate::OnCancelURLRequestWithPolicyViolatingReferrerHeader(
   return false;
 }
 
-// TODO(deepak1556) : Enable after hooking into the reporting service
-// https://crbug.com/704259
 bool AtomNetworkDelegate::OnCanQueueReportingReport(
     const url::Origin& origin) const {
-  return false;
+  return true;
 }
 
 void AtomNetworkDelegate::OnCanSendReportingReports(
     std::set<url::Origin> origins,
-    base::OnceCallback<void(std::set<url::Origin>)> result_callback) const {}
+    base::OnceCallback<void(std::set<url::Origin>)> result_callback) const {
+  std::move(result_callback).Run(std::move(origins));
```

Comment:
Will we be exposing this to users to implement with some permission model ?

Reply:
hm... yes, i'd like to, but i'm not sure where to expose the method. WebRequest doesn't seem like the right place. Session doesn't seem to have access to the network delegate currently. Thoughts?

Reply:
I think its better to wait for the webrequest and network delegate refactor with the network service to expose this properly. In the meantime , can you leave a `TODO`